roles/adoption_osp_deploy/tasks/prepare_undercloud.yml

```diff
@@ -54,6 +54,17 @@
         script: "{{ _container_prepare_cmd }}"
       when: cifmw_adoption_osp_deploy_scenario.container_prepare_params is not defined
 
+    # Adoption requires Ceph 7 (Reef) as a requirement. Instead of performing a Ceph
```

Comment:
@cescgina this is supposed to be the equivalent of https://github.com/openstack-k8s-operators/install_yamls/pull/930.

However, this is not enough, because due to a bz that I will link, I need to change something else in the TripleO task (see https://github.com/openstack-k8s-operators/install_yamls/pull/930).

Where is deployed Ceph in this role?

Reply:
@fmount thanks for looking into this. This playbook https://github.com/openstack-k8s-operators/ci-framework/blob/main/hooks/playbooks/adoption_deploy_ceph.yml deploys ceph in this role (it's called as a hook)

Reply:
Thank you! I'm going to update that playbook in my next iteration to this patch!